corehq/apps/users/models.py

```diff
@@ -3143,6 +3143,12 @@ def deactivate_mobile_workers(cls, domain, date_deactivation):
     @classmethod
     def update_trigger(cls, domain, user_id, deactivate_after):
         existing_trigger = cls.objects.filter(domain=domain, user_id=user_id)
+        if not deactivate_after:
```

Comment:
A test for _`deactivate_after` is a blank string_ would be nice since https://github.com/dimagi/commcare-hq/pull/31180/commits/1319dc06ea945b3e7b367d43f286b3c5d67bb68e appears to be a fix aimed at that scenario.

Reply:
291d4260e05c4a48a3080a2c5b8b028f7a2ffa98